src/locales/en/fields.yml

```diff
@@ -8,25 +8,33 @@
   label: Year
 
 surname:
-  label: Last Name
+  label: Last name
   hint:
   validation:
     default: "Enter your last name as it appears on your photo ID"
     maxlength: "Enter your last name as it appears on your photo ID"
-    regexName: "Enter your last name as it appears on your photo ID"
+    regexNumber: "Your last name cannot include numbers"
+    regexSpecialCharacters: "Enter your last name as it appears on your photo ID"
+
 
 firstName:
   label: First name
   hint:
   validation:
     default: "Enter your first name as it appears on your photo ID"
     maxlength: "Enter your first name as it appears on your photo ID"
-    regexName: "Enter your first name as it appears on your photo ID"
+    regexNumber: "Your first name cannot include numbers"
+    regexSpecialCharacters: "Enter your first name as it appears on your photo ID"
 
 middleName:
-  label: Middle names
+  label: Middle name(s)
   hint: Leave this blank if you do not have any middle names
-  regexName: "Enter your first name as it appears on your photo ID"
+  validation:
+    default: "Enter your middle names as they appear on your photo ID"
+    maxlength: "Enter your first names as they appear on your photo ID"
```

Comment:
```suggestion
    maxlength: "Enter your first names as they appear on your photo ID"
    maxlength: "Enter your middle names as they appear on your photo ID"
```

Reply:
Do we want something similar for last name as well? So it would be `Enter your last names as they appear on your photo ID` instead of `Enter your last name as it appears on your photo ID`?

Reply:
https://gds.slack.com/archives/C0440GZHUUX/p1690368329573179?thread_ts=1690304173.634379&cid=C0440GZHUUX